src/datastore/common.ts

```diff
@@ -328,6 +328,11 @@ export interface DataStore extends DataStoreEventEmitter {
   }): Promise<{ results: DbBlock[]; total: number }>;
   getBlockTxs(indexBlockHash: string): Promise<{ results: string[] }>;
   getBlockTxsRows(blockHash: string): Promise<FoundOrNot<DbTx[]>>;
+  getTxsFromBlock(
+    indexBlockHash: string,
```

Comment:
This should be `blockHash` which is the STX block hash. `indexBlockHash` is a combination of the STX and BTC block.